source/Weapon.h

```diff
@@ -420,6 +425,7 @@ inline bool Weapon::IsSafe() const { return isSafe; }
 inline bool Weapon::IsPhasing() const { return isPhasing; }
 inline bool Weapon::IsDamageScaled() const { return isDamageScaled; }
 inline bool Weapon::IsGravitational() const { return isGravitational; }
+inline bool Weapon::IsExplodeOnDeath() const { return isExplodeOnDeath; }
```

Comment:
I'd rename this to ExplodesOnDeath() honestly, we dont have IsBlastRadius

Reply:
Well blast radius is a double, not a boolean.